arrow/src/test/java/org/apache/calcite/adapter/arrow/ArrowAdapterTest.java

```diff
@@ -325,23 +325,15 @@ static void initializeArrowState(@TempDir Path sharedTempDir) throws IOException
         .explainContains(plan);
   }
 
+  /** <a href="https://issues.apache.org/jira/browse/CALCITE-6296">[CALCITE-6296]</a>. */
   @Test void testArrowProjectFieldsWithIsNullFilter() {
```

Comment:
Maybe we can add another test for "a is null and b is null" or "a is null or b=c", To cover when the IS NULL is in a complicated condition.

Reply:
Good point! Added tests with [Add Conjunctive IsNull Filter tests](https://github.com/apache/calcite/pull/3858/commits/6d31f70295e36529ff5691ae1d52f1a4c95389c8)